packages/infra/lib/ihe-stack.ts

```diff
@@ -86,6 +86,55 @@ export class IHEStack extends Stack {
       },
     });
 
+    // Create lambdas
+    const iti38Lambda = createLambda({
+      stack: this,
+      name: "ITI38",
+      entry: "iti38",
+      layers: [lambdaLayers.shared],
+      envType: props.config.environmentType,
+      envVars: {
+        ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
+      },
+      vpc: props.vpc,
+      alarmSnsAction: props.alarmAction,
+    });
+
+    const iti39Lambda = createLambda({
+      stack: this,
+      name: "ITI39",
+      entry: "iti39",
+      layers: [lambdaLayers.shared],
+      envType: props.config.environmentType,
+      envVars: {
+        ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
+      },
+      vpc: props.vpc,
+      alarmSnsAction: props.alarmAction,
+    });
+
+    const iti55Lambda = createLambda({
+      stack: this,
+      name: "ITI55",
+      entry: "iti55",
+      layers: [lambdaLayers.shared],
+      envType: props.config.environmentType,
+      envVars: {
+        ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
+      },
+      vpc: props.vpc,
+      alarmSnsAction: props.alarmAction,
+    });
+
+    // Create resources for each lambda directly under the API root
+    const iti38Resource = api.root.addResource("ITI38");
```

Comment:
Beyond renaming to domain/business operations, we don't use caps for http resources